views/default/tab_client_snapshots.pdt

```diff
@@ -13,27 +13,29 @@
                 <tr>
                     <th><?php $this->_('Vultr.tab_client_snapshots.snapshot_id'); ?></th>
                     <th><?php $this->_('Vultr.tab_client_snapshots.description'); ?></th>
+                    <th><?php $this->_('Vultr.tab_client_snapshots.status'); ?></th>
                     <th><?php $this->_('Vultr.tab_client_snapshots.options'); ?></th>
                 </tr>
             </thead>
             <tbody>
                 <?php
-                foreach ($snapshots as $snapshotid => $description) {
+                foreach ($snapshots as $snapshot) {
                 ?>
                 <tr>
-                    <td><strong><?php $this->Html->_($snapshotid); ?></strong></td>
-                    <td><?php $this->Html->_($description); ?></td>
+                    <td><strong><?php $this->Html->_($snapshot->SNAPSHOTID); ?></strong></td>
+                    <td><?php $this->Html->_($snapshot->description); ?></td>
+                    <td><?php echo ucwords($this->Html->safe($snapshot->status)); ?></td>
                     <td>
-                        <a class="btn btn-xs btn-success restore-snapshot" href="#" data-id="<?php $this->Html->_($snapshotid); ?>" data-toggle="modal" data-target="#restore-snapshot<?php echo $snapshotid; ?>">
+                        <a class="btn btn-xs btn-success restore-snapshot" href="#" data-id="<?php $this->Html->_($snapshot->SNAPSHOTID); ?>" data-toggle="modal" data-target="#restore-snapshot<?php echo $snapshot->SNAPSHOTID; ?>">
```

Comment:
The $snapshot->SNAPSHOTID should be HTML-safe like in other locations

Reply:
It is